src/pages/public_registration/PublicSummaryContent.tsx

```diff
@@ -1,104 +1,44 @@
-      return matchingActivity ? getLanguageString(matchingActivity.label) : '';
-    });
-
-  const selectedHrcsCategories = subjects
-    .filter((subjectId) => subjectId.startsWith(hrcsCategoryBaseId))
-    .map((categoryId) => {
-      const matchingCategory = hrcsCategories.categories.find((category) => category.id === categoryId);
-      return matchingCategory ? getLanguageString(matchingCategory.label) : '';
-    });
+  const { entityDescription } = registration;
 
-  return (
+  return !entityDescription ? null : (
     <>
-      {entityDescription && (
-        <>
-          {entityDescription.abstract && (
-            <Typography style={{ whiteSpace: 'pre-line' }}>{entityDescription.abstract}</Typography>
-          )}
-          {entityDescription.alternativeAbstracts.und && (
-            <Accordion elevation={0}>
-              <AccordionSummary expandIcon={<ExpandMoreIcon color="primary" />} sx={{ padding: '0' }}>
-                <Typography variant="h3" color="primary">
-                  {t('registration.description.alternative_abstract')}
-                </Typography>
-              </AccordionSummary>
-              <AccordionDetails sx={{ padding: 0 }}>
-                <Typography style={{ whiteSpace: 'pre-line' }} paragraph>
-                  {entityDescription.alternativeAbstracts.und}
-                </Typography>
-              </AccordionDetails>
-            </Accordion>
-          )}
-          {entityDescription.description && (
-            <>
-              <Typography variant="h3" color="primary">
-                {t('common.description')}
-              </Typography>
-              <Typography style={{ whiteSpace: 'pre-line' }} paragraph>
-                {entityDescription.description}
-              </Typography>
-            </>
-          )}
-
-          {entityDescription.tags.length > 0 && (
-            <TagsList title={t('registration.description.keywords')} values={entityDescription.tags} />
-          )}
-        </>
+      {entityDescription.abstract && (
+        <Typography style={{ whiteSpace: 'pre-line' }} paragraph>
+          {entityDescription.abstract}
+        </Typography>
       )}
-
-      {selectedHrcsActivities.length > 0 && (
-        <TagsList title={t('registration.description.hrcs_activities')} values={selectedHrcsActivities} />
+      {entityDescription.alternativeAbstracts.und && (
+        <Accordion elevation={0}>
+          <AccordionSummary expandIcon={<ExpandMoreIcon color="primary" />} sx={{ padding: '0' }}>
+            <Typography variant="h3" color="primary">
+              {t('registration.description.alternative_abstract')}
+            </Typography>
+          </AccordionSummary>
+          <AccordionDetails sx={{ padding: 0 }}>
+            <Typography style={{ whiteSpace: 'pre-line' }} paragraph>
+              {entityDescription.alternativeAbstracts.und}
+            </Typography>
+          </AccordionDetails>
+        </Accordion>
```

Comment:
Virker rart med en accordion her IMO, så avventer svar på om den skal beholdes eller ikke: https://sikt-no.slack.com/archives/CPHGQRNAJ/p1688051909450149

Reply:
en eventuell endringer der kan for så vidt like godt tas i egen PR

Reply:
Løses i https://github.com/BIBSYSDEV/NVA-Frontend/pull/4243